schemas/com.mparticle.snowplow/pushregistration_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an push registration event",
+	"self": {
+		"vendor": "com.mparticle.snowplow",
+		"name": "pushregistration_event",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"name": {
+			"type": "string"
+		},
+		"registrationToken": {
+			"type": "string"
+		},
+		"attributes": {}
```

Comment:
What does this mean?

Reply:
This is meant to be a dictionary for any key value pairs. Seems like we had it for our AppEvent in our V1: https://github.com/snowplow/iglu-central/blob/master/schemas/com.mparticle.snowplow/app_event/jsonschema/1-0-0 so I assume this should be fine.

Reply:
Cool, thanks for clarifying